packages/data-context/src/actions/WizardActions.ts

```diff
@@ -207,14 +209,18 @@ export class WizardActions {
     await this.ctx.fs.mkdir(supportDir, { recursive: true })
 
     let fileContent: string | undefined
+    let description: string = ''
 
     if (fileName === 'commands') {
       fileContent = commandsFileBody(language)
+      description = 'A support file that is useful for creating custom Cypress commands and overwriting existing ones.'
```

Comment:
Ideally these strings would be set by the frontend instead of the backend so we can store them in `en-US.json`, but doing so would require some backend changes that would have significantly bloated the scope for this ticket.
Would be great to do at another time